keylime/cli/options.py

```diff
@@ -29,4 +31,4 @@ def get_opts_error(args):
         return True, "--allowlist-sig-url must also have --allowlist-sig-key"
     if args.allowlist_sig_key and not (args.allowlist_sig or args.allowlist_sig_url):
         return True, "--allowlist-sig-key must have either --allowlist-sig or --allowlist-sig-url"
-    return False, None
+    return False, ""
```

Comment:
Why go with an empty string vs None here?

Reply:
Because of the annotated type (`-> Tuple[bool, str]`)